contrib/test/integration/vars.yml

```diff
@@ -3,6 +3,24 @@
 # When swap setup is necessary, make it this size
 swapfileGB: 8
 
+# Path to encode into /etc/environment on all hosts
+go_path: "/go"
+
+# Absolute path on control-host where the cri-o source exists
+cri_o_src_path: "{{ playbook_dir }}/../../../"
+
+# Absolute path on subjects where cri-o source is expected
+cri_o_dest_path: "{{ go_path }}/src/github.com/kubernetes-incubator/cri-o"
+
+# Global environment variables to use for all tasks
```

Comment:
But these aren't global vars that we use for all tasks .. they were mostly very specific things that need to be set for one task or two...

Reply:
Yes,  originally tried just applying ``environment`` in specific tasks, but that makes it really hard to debug - since they're missing from the Ansible output. So I thought it made more sense to just make them global, and if that causes specific problems, then individual items can be overridden by task-local ``environment`` directives (I tested this, and it works).

That said, I'm more than willing to compromise on this if you have a better idea.  My goal was to help reduce mental-load by allowing engineers to just assume one set of env. vars are set everywhere, all the time.  But maybe there's some easier way?